config/traefik/traefik.yml

```diff
@@ -23,3 +23,8 @@ log:
 global:
   checkNewVersion: false
   sendAnonymousUsage: false
+experimental:
```

Comment:
If PR #813 is approved. This can be refactored into `docker/docker-compose.sablier.yml` as environment variables. Preventing the need to install the plugin even if Sablier is disabled.